portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,9 +35,14 @@
   private final List<String> myComments = new ArrayList<>();
   private final DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
 
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment");
```

Comment:
Since we added timestamp in the parent PR - can we use that to sort here (per the exercise)?

Reply:
yes! will do!